src/azure-cli-core/azure/cli/core/_session.py

```diff
@@ -38,7 +44,7 @@ def load(self, filename, max_age=0):
                     self.save()
             with codecs_open(self.filename, 'r', encoding=self._encoding) as f:
                 self.data = json.load(f)
-        except (OSError, IOError):
+        except (OSError, IOError, t_JSONDecodeError):
```

Comment:
I would suggest treat this warning differently by outputting a warning which tells people we are doing this, as the error could be caused by non empty but bad file and users might be surprised to see all the accounts are lost now

Reply:
Will do. However, currently `OSError` and `IOError` are already doing this.